assets/js/phoenix_live_view.js

```diff
@@ -468,10 +476,32 @@ export class LiveSocket {
     }
   }
 
-  bindClicks(){
-    window.addEventListener("click", e => {
+  bindTouchstarts(){
+    window.addEventListener("touchstart", e => {
```

Comment:
It could, I guess, but I chose not to over engineer it for now, because
1. usually people use live_view to replace client-side heavy javascript, the limited custom event listeners of "window on touchstart" (or "window on touchmove") should ease the perf impacts.
1. scrolling works on my app without any lagging.